config/software/datadog-process-agent.rb

```diff
@@ -0,0 +1,15 @@
+name "datadog-process-agent"
+always_build true
+
+build do
+  agent_branch = ENV['PROCESS_AGENT_BRANCH']
+  if agent_branch.nil? || agent_branch.empty?
+    agent_branch = "master"
+  end
+
+  binary = "process-agent-amd64-#{agent_branch}"
```

Comment:
And you probably want to use the `ship_license` omnibus method to make sure we ship the corresponding license.